llvm/lib/Transforms/Yk/LivenessAnalysis.cpp

```diff
@@ -40,7 +40,7 @@ void vset_union(const std::set<Value *> &S1, const std::set<Value *> &S2,
 }
 
 namespace llvm {
-/// Find the successor instructions of the specified instruction.
```

Comment:
This shouldn't have been deleted shouldit?

Reply:
In the commit message i talked about removing the duplication of the docstrings based on an answer to  stackoverflow question (found by searching for "should i document c++ in the header or the cpp file :P).

Reply:
Ah I see what that means now!